### DISCUSSION TITLE
cmake error

### DISCUSSION THREAD

**Discussion by dreamkeep** [zephyrproject-rtos/zephyr#52400] [Q&A]
Describe the bug
I newly install windows 8.1  on my pc, and follow the start guide, below is the error i meet.

PIO Core Call Error: "The current working directory H:\\work-dir\\zephyr\\zephyr-blink will be used for the project.\r\n\r\nThe next files/directories have been created in H:\\work-dir\\zephyr\\zephyr-blink\r\ninclude - Put project header files here\r\nlib - Put here project specific (private) libraries\r\nsrc - Put project source files here\r\nplatformio.ini - Project Configuration File\r\nResolving nrf52_dk dependencies...\r\nAlready up-to-date.\r\nUpdating metadata for the vscode IDE...\r\n\n\nError: Processing nrf52_dk (platform: nordicnrf52; board: nrf52_dk; framework: zephyr)\r\n--------------------------------------------------------------------------------\r\nVerbose mode can be enabled via `-v, --verbose` option\r\nCONFIGURATION: https://docs.platformio.org/page/boards/nordicnrf52/nrf52_dk.html\r\nPLATFORM: Nordic nRF52 (9.4.0) > Nordic nRF52-DK\r\nHARDWARE: NRF52832 64MHz, 64KB RAM, 512KB Flash\r\nDEBUG: Current (jlink) On-board (cmsis-dap, jlink) External (blackmagic, stlink)\r\nPACKAGES: \r\n - framework-zephyr @ 2.20701.220422 (2.7.1) \r\n - tool-cmake @ 3.21.3 \r\n - tool-dtc @ 1.4.7 \r\n - tool-ninja @ 1.9.0 \r\n - tool-sreccat @ 1.164.0 (1.64) \r\n - toolchain-gccarmnoneeabi @ 1.80201.190214 (8.2.1)\r\nReading CMake configuration...\r\n-- Application: H:/work-dir/zephyr/zephyr-blink/zephyr\r\n-- Zephyr version: 2.7.1 (C:/Users/Administrator/.platformio/packages/framework-zephyr)\r\n-- Found Python3: C:/Users/Administrator/.platformio/penv/Scripts/python.exe (found suitable exact version \"3.9.10\") found components: Interpreter \r\n-- Board: nrf52dk_nrf52832\r\n-- Cache files will be written to: C:/Users/Administrator/.platformio/packages/framework-zephyr/.cache\r\n-- Found dtc: C:/Users/Administrator/.platformio/packages/tool-dtc/dtc.exe (found suitable version \"1.4.7\", minimum required is \"1.4.6\")\r\n-- Found toolchain: gnuarmemb (C:/Users/Administrator/.platformio/packages/toolchain-gccarmnoneeabi)\r\n-- Found BOARD.dts: C:/Users/Administrator/.platformio/packages/framework-zephyr/boards/arm/nrf52dk_nrf52832/nrf52dk_nrf52832.dts\r\n-- Configuring incomplete, errors occurred!\r\n\r\narm-none-eabi-gcc.exe: error: CreateProcess: No such file or directory\r\nCMake Error at C:/Users/Administrator/.platformio/packages/framework-zephyr/cmake/dts.cmake:167 (message):\r\n command failed with return code: 1\r\nCall Stack (most recent call first):\r\n C:/Users/Administrator/.platformio/packages/framework-zephyr/cmake/app/boilerplate.cmake:541 (include)\r\n CMakeLists.txt:2 (include)\r\n\r\n\r\n========================== [FAILED] Took 2.15 seconds =========================="

**Comment by stephanosio** [comment]
We do not support PlatformIO. Please ask any PlatformIO questions on the relevant PlatformIO forums.

**Reply by dreamkeep** [reply]
ok,tks.
